presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add ``ngrams`` function to generate the N-grams from an array.
+* Add ``qdigest`` type and associated functions.
+* Add statistics calculator support for ``TINYINT``, ``SMALLINT`` and ``VARBINARY`` literals.
+* Add system to delay query execution until a minimum number of workers nodes are available. If the requirement is not met within a time limit, the query is failed.
+* Improve error message for broken :func:``array_sort`` comparator.
+* Fix memory leak related to query tracking in coordinator. Completed queries are supposed to be purged shortly after its completion. However, a 0.213 regression broke this.
+* Lazily build hash table for ``MapBlock`` when the hash table is used in ``SingleMapBlock``.
+* Make planning more stable with regard to query predicates and joins.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield signals.
```

Comment:
What does making “planning more stable” mean in this context? What’s the user-visible effect?

Reply:
If you write filter `a = b AND b = c` it won't be reordered to `b = c AND a = b` as often as previously. If it's too vague we can remove it

Reply:
From our example, the outcome is to produce a query plan that matches user's expectations by being consistent with the order of predicates and joins in the original SQL statement. It has performance implication if the author of the query expected to short-circuit more often in the order they write the predicate.

What about:

```
* Maintain order of predicates and join tables in the query plan
```
?